test/times-table-test.js

```diff
@@ -0,0 +1,16 @@
+const times = require('../lib/times-table')
+const expect = require('chai').expect
+
+describe('times-table functions', function (){
+  it('createArray function returns array with numbers', function(){
+    expect(times.createArray(5)).to.eql([1, 2, 3, 4, 5])
+  })
+  it('generateRows  returns array of arrays for table rows', function(){
```

Comment:
i'd have a separate describe block for each function that you'll testing - this will help organize your tests and make them easier to read 👍